aspnetcore/migration/22-to-30.md

```diff
@@ -437,35 +437,45 @@ The most significant change from `WebHostBuilder` to `HostBuilder` is in [depend
 
 ## Update SignalR code
 
-If you call `AddJsonProtocol`, replace it with `AddNewtonsoftJsonProtocol`.
+System.Text.Json is now the default Hub Protocol used by both the client and server.
 
-* The following examples show server code before and after the change:
+Use `AddJsonProtocol` to set serializer options.
 
-  ```csharp
-  services.AddSignalR(...)
-          .AddJsonProtocol(...) // 2.2
-  ```
-
-  ```csharp
-  services.AddSignalR(...)
-          .AddNewtonsoftJsonProtocol(...) // 3.0
-  ```
+Server:
+```csharp
+services.AddSignalR(...)
+        .AddJsonProtocol(options =>
+        {
+            options.WriteIndented = false;
```

Comment:
We should link to the AddJsonProtocol/JsonHubProtocolOptions API docs once they're updated.